src/qtcwin.c

```diff
@@ -0,0 +1,1814 @@
+	    }
+	    refreshp();
+	    x = onechar();
+	}
+
+	switch(x) {
+	    case 152:	// cursor up
+		      wattrset(ry_win, line_normal);
+		      print_rtty_line(qtc_ry_lines[actline-1], actline);
+		      if (actline > 1) {
+			  actline--;
+		      }
+		      else {
+			  if (qtc_ry_currline > 0) {
+			      actline = qtc_ry_currline;
+			  }
+		      }
+		      wattrset(ry_win, line_currnormal);
+		      print_rtty_line(qtc_ry_lines[actline-1], actline);
+		      wattrset(ry_win, line_normal);
+		      break;
+	    case 153:	// cursor down
+		      wattrset(ry_win, line_normal);
+		      print_rtty_line(qtc_ry_lines[actline-1], actline);
+		      if (actline < qtc_ry_currline) {
+			  actline++;
+		      }
+		      else {
+			  actline = 1;
+		      }
+		      wattrset(ry_win, line_currnormal);
+		      print_rtty_line(qtc_ry_lines[actline-1], actline);
+		      wattrset(ry_win, line_normal);
+		      break;
+	    case 18:	// CTRL-r
+		      for(j=0; j<12; j++) {
+			  qtc_ry_lines[j].content[0] = '\0';
+			  qtc_ry_lines[j].attr = 0;
+			  mvwprintw(ry_win, (j+1), 1, "                                      ");
+		      }
+		      prevline = -1;
+		      actline = 1;
+		      qtc_ry_currline = 0;
+		      qtc_ry_copied = 0;
+		      break;
+	    case 19:	// CTRL-s  -  start capture
+		      qtc_ry_capture = 1;
+		      mvwprintw(qtcwin, 2, 11, "CAPTURE ON ");
+		      break;
+	    case 5:	// CTRL-e  -  end capture
+		      qtc_ry_capture = 0;
+		      mvwprintw(qtcwin, 2, 11, "CAPTURE OFF");
+		      break;
+	    case 10:	// ENTER  -  add to qtc
+		      if (qtc_ry_lines[actline-1].attr == 0) {
+			  parse_ry_line(qtc_ry_lines[actline-1].content);
+			  qtc_ry_lines[actline-1].attr = 1;
+		      }
+		      wattrset(ry_win, line_currnormal);
+		      print_rtty_line(qtc_ry_lines[actline-1], actline);
+		      wattrset(ry_win, line_normal);
+		      break;
+	}
+	refreshp();
+	if (x != 27) {
+	    x = -1;
+	}
+	if (*qtccount > 0 && qtc_ry_copied == *qtccount) {
+	    qtc_ry_capture = 0;
+	    mvwprintw(qtcwin, 2, 11, "CAPTURE OFF");
+	}
+	nodelay(stdscr, FALSE);
+    }
+    hide_panel(ry_panel);
+    hide_panel(ry_help_panel);
+    //mvwprintw(qtcwin, 2, 11, "           ");
+
+    curs_set(1);
+    tactivefield = activefield;
+    activefield = oactivefield;
+    showfield(tactivefield);
+    showfield(oactivefield);
+    
+    return 0;
+}
+
+int put_qtc() {
+
+    init_pair(QTCRECVLINE,    COLOR_WHITE,  COLOR_BLUE);
+    int line_normal = COLOR_PAIR(QTCRECVLINE) | A_NORMAL;
+    char qtcdirstring[3][10] = {"", "Received", "Sent"};
+
+    wattrset(qtcwin, line_normal);
+    if (*qtccurrdiretion == RECV || *qtccurrdiretion == SEND) {
+	mvwprintw(qtcwin, 1, 19, "%s %2d QTC", qtcdirstring[*qtccurrdiretion], qtc_temp_obj->total);
+    }
+    else {
+	mvwprintw(qtcwin, 1, 19, "Total: %2d QTC (R: %d, S: %d)", qtc_temp_obj->total, qtc_temp_obj->received, qtc_temp_obj->sent);
+    }
+    return 0;
+}
+
+int strip_spaces(char * src, char * tempc) {
```

Comment:
I've changed to replace_spaces(). Now I keept the own function instead of the g_str*() functions. Pushed.